projects/lf-documentation/src/app/getting-started/getting-started.component.html

```diff
@@ -13,12 +49,13 @@
       <span>=</span>
       <span class="code-html-attribute-value"
         >"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/laserfiche-ui-components.js"
-      </span>     <span class="code-html-attribute-name">defer</span>
+      </span>
+      <span class="code-html-attribute-name">defer</span>
       <span class="code-html-tagname">&gt;&lt;/script&gt;</span>
     </code>
     <br /><br />
   </p>
-  <p class="documentation-text">Choose a theme to style your components:</p>
+  <p class="documentation-text">Choose a theme to style your components. Import the style sheet from the CDN by adding this line to your html file inside your application:</p>
```

Comment:
Import the style sheet from the CDN by adding this line to the html file inside your application. You must choose either lf-laserfiche-lite or lf-microsoft-lite
(and show both scripts)